sockjs/frame.go

```diff
@@ -1,11 +1,41 @@
 package sockjs
 
 import (
-	"encoding/json"
-	"fmt"
+	"bytes"
+	"strconv"
 )
 
 func closeFrame(status uint32, reason string) string {
-	bytes, _ := json.Marshal([]interface{}{status, reason})
-	return fmt.Sprintf("c%s", string(bytes))
+	return "c[" + strconv.FormatUint(uint64(status), 10) + "," + strconv.Quote(reason) + "]"
+}
+
+func frame(messages []string) string {
+	switch len(messages) {
+	case 0:
+		return ""
+	case 1:
+		return messageFrame(messages[0])
+	default:
+		return arrayFrame(messages)
+	}
+}
+
+func messageFrame(message string) string {
+	return "m" + strconv.Quote(message)
```

Comment:
I was able to reduce your sample to this input:

```json
{
  "寔愆샠5]䗄IH贈=d﯊/偶?ॊn%晥D視N򗘈'᫂⚦|X쵩넽z질tskxDQ莮Aoﱻ뛓": true
}
```

It makes json encoder fail with:

```go
&json.SyntaxError{msg:"invalid character 'U' in string escape code", Offset:61}
```

The offset points to this sequence `\U00097608`.

I will bring back json.Marshal and add this as a testcase.